Frontend/src/pages/registrations/components/RegistrationList.jsx

```diff
@@ -18,15 +65,157 @@ export default function RegistrationList() {
+                    className="registrations-table-header-item"
+                  >
+                    <CubingIcon event={id} selected />
+                  </Table.HeaderCell>
+                ))}
+                <Table.HeaderCell
+                  className="registrations-table-header-item"
+                  sorted={column === 'total' ? direction : undefined}
+                  onClick={() =>
+                    dispatch({ type: 'CHANGE_SORT', column: 'total' })
+                  }
+                >
+                  Total
+                </Table.HeaderCell>
+              </Table.Row>
+            </Table.Header>
+            <Table.Body>
+              {data ? (
+                data.map((registration) => (
+                  <Table.Row
+                    key={`registration-table-row-${registration.user.id}`}
+                  >
+                    <Table.Cell>
+                      {registration.user.wca_id ? (
+                        <a
+                          href={`https://worldcubeassociation.org/persons/${registration.user.wca_id}`}
+                        >
+                          {registration.user.name}
+                        </a>
+                      ) : (
+                        registration.user.name
+                      )}
+                    </Table.Cell>
+                    <Table.Cell>
+                      <FlagIcon iso2={registration.user.country.iso2} />
+                      {registration.user.country.name}
+                    </Table.Cell>
+                    {competitionInfo.event_ids.map((id) => {
+                      if (registration.competing.event_ids.includes(id)) {
+                        return (
+                          <Table.Cell
+                            key={`registration-table-row-${registration.user.id}-${id}`}
+                          >
+                            <CubingIcon event={id} selected={true} />
+                          </Table.Cell>
+                        )
+                      }
+                      return (
+                        <Table.Cell
+                          key={`registration-table-row-${registration.user.id}-${id}`}
+                        />
+                      )
+                    })}
+                    <Table.Cell>
+                      {registration.competing.event_ids.length}
+                    </Table.Cell>
+                  </Table.Row>
+                ))
+              ) : (
+                <Table.Row>
+                  <Table.Cell width={12}> No matching records found</Table.Cell>
+                </Table.Row>
+              )}
+            </Table.Body>
+            <Table.Footer>
+              <Table.Row>
+                <Table.Cell>{`${newcomers} First-Timers + ${
+                  registrations.length - newcomers
+                } Returners = ${registrations.length} People`}</Table.Cell>
+                <Table.Cell>{`${countrySet.size}  Countries`}</Table.Cell>
```

Comment:
Is this one of those instances of 'country' that need to be changed to 'region' (potentially as a separate PR)?

Reply:
These changes will all be done as part of integrating i18n